store/src/main/java/org/apache/rocketmq/store/CommitLog.java

```diff
@@ -802,8 +802,8 @@ private String generateKey(StringBuilder keyBuilder, MessageExt messageExt) {
         PutMessageResult putMessageResult = new PutMessageResult(PutMessageStatus.PUT_OK, result);
 
         // Statistics
-        storeStatsService.getSinglePutMessageTopicTimesTotal(messageExtBatch.getTopic()).addAndGet(result.getMsgNum());
-        storeStatsService.getSinglePutMessageTopicSizeTotal(messageExtBatch.getTopic()).addAndGet(result.getWroteBytes());
+        storeStatsService.getSinglePutMessageTopicTimesTotal(messageExtBatch.getTopic()).add(1);
```

Comment:
Why use '1' instead of 'result.getMsgNum()'

Reply:
save a method invocation

Reply:
Your are right, i will fix it

Reply:
fixed